packages/web-react/src/components/Grid/README.md

```diff
@@ -54,15 +54,54 @@ Custom vertical (y-axis) spacing:
 </Grid>
 ```
 
+## Item Alignment
+
+The `alignmentX` and `alignmentY` props are used to control the alignment of items within a container.
+The available values for these properties can be found in our [alignment dictionary][alignment-dictionary].
+
+`alignmentX`: Manages horizontal alignment, allowing you to position items to the left, center, or right of the container. It can also be configured with responsive values for different breakpoints.
+`alignmentY`: Manages vertical alignment, enabling you to position items at the top, center, or bottom of the container. It supports responsive values for various breakpoints as well.
+
+Both props can be set using either fixed values or objects with breakpoint-specific settings to ensure the alignment adapts across different screen sizes.
+
+Horizontal Alignment:
+
+```jsx
+<Grid alignmentX="left">
+  <!-- Grid content -->
+</Grid>
+```
+
+Horizontal and Vertical Alignment:
+
+```jsx
+<Grid alignmentX="left" alignmentY="top">
+  <!-- Grid content -->
+</Grid>
+```
+
+Responsive Horizontal and Vertical Alignment:
+
+```jsx
+<Grid
+  alignmentX={{ mobile: 'left', tablet: 'center', desktop: 'right' }}
+  alignmentY={{ mobile: 'top', tablet: 'center', desktop: 'bottom' }}
+>
+  <!-- Grid content -->
+</Grid>
+```
+
```

Comment:
Please point out the caveats of the `stretch` option, something like:

```suggestion
👉 Please note that the `stretch` option may have undesired impact on elements with a defined size or aspect ratio. The dimensions (or the ratio) will not respected by `Grid` and the element will be stretched just like any other item.
```

cc @crishpeen